packages/@aws-cdk/aws-ecs/README.md

```diff
@@ -335,6 +335,35 @@ const service = new ecs.FargateService(this, 'Service', {
 `Services` by default will create a security group if not provided.
 If you'd like to specify which security groups to use you can override the `securityGroups` property.
 
+### Deployment circuit breaker and rollback
+
+By default, Amazon ECS [deployment circuit breaker](https://aws.amazon.com/tw/blogs/containers/announcing-amazon-ecs-deployment-circuit-breaker/)
+is enabled and will rollback on deployment failure. With this feature, Amazon ECS will automatically roll back unhealthy
+service deployments without the need for manual intervention. As this feature with AWS Fargate requires
+platform `1.4.0` or above. Make sure to specify the `platformVersion` to `1.4` as the current default version points to `1.3`.
```

Comment:
I don't think this is correct, we've tried circuit breaker with platform version 1.3. The documentation doesn't mention this being a requirement.

Reply:
> Is this correct? The documentation doesn't mention this being a requirement afaik.

Thanks for bringing it up. I'll confirm this internally.